Sources/XCTestDynamicOverlay/Internal/XCTCurrentTestCase.swift

```diff
@@ -0,0 +1,31 @@
+#if DEBUG
+  #if canImport(ObjectiveC)
+    import Foundation
+
+    var XCTCurrentTestCase: AnyObject? {
```

Comment:
I factored this out thinking I'd use it for the logic in `XCTIsTesting`, but decided it wasn't needed. Still kinda handy to have in one place, though, especially for exploring dynamic expectations, as well...